src/poker_kata/poker.clj

```diff
@@ -10,15 +10,21 @@
 ;------------ Utils to transform string to card object
 
 (defn card-value-number [val]
-  (case (string/lower-case val)
-    "a" 14
-    "j" 11
-    "q" 12
-    "k" 13
-    (parse-int val)))
+  (let [card-value-map {"a" 14
+                        "j" 11
+                        "q" 12
+                        "k" 13}]
+    (-> (get card-value-map (string/lower-case val))
```

Comment:
``` clojure
  (def card->val  {"j" 11, "q" 12, "k" 13, "a" 14})
  (defn card-value-number* [card-value]
      (card->val card-value (parse-int card-value)))
  (def card-value-number (comp card-value-number* string/lower-case))
```

Reply:
function get, using keyword or hashmap as a function  all have the last argument that accepts default value when getting nil

Reply:
Nice!!